service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseManagementOrder.java

```diff
@@ -23,4 +23,5 @@
     private final List<Element<Recital>> recitals;
     private final CMOStatus cmoStatus;
     private final DocumentReference orderDoc;
+    private final CaseManagementOrderAction caseManagementOrderAction;
```

Comment:
ok I mind this being like this less now. CMOStatus should be OrderStatus. Maybe this isn't place for you story. When the judge says no or says yes, the status here should update